app/src/main/res/layout/fragment_skill_details.xml

```diff
@@ -376,6 +376,64 @@
                 android:layout_marginTop="@dimen/margin_small"
                 android:visibility="gone" />
 
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:text="@string/feedback"
+                android:textColor="@color/md_black_1000"
+                android:textSize="@dimen/text_size_normal" />
+
+            <TextView
+                android:id="@+id/tvPostFeedbackDesc"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/desc_post_feedback"
+                android:visibility="gone" />
+
+            <LinearLayout
+                android:id="@+id/layoutPostFeedback"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:visibility="gone">
+
+                <android.support.design.widget.TextInputLayout
+                    android:id="@+id/feedback"
+                    android:layout_width="0dp"
```

Comment:
Use dimen resource

Reply:
When only a single layout in a LinearLayout defines a weight, it is more efficient to assign a width/height of 0dp to it since it will absorb all the remaining space anyway. With a declared width/height of 0dp it does not have to measure its own size first.
Also, 0dp remains the same for all screen sizes.

Reply:
it does not define why it should not be in dimen resource

Reply:
Android Studio itself points out not to use it. 
<img width="1204" alt="screen shot 2018-07-03 at 1 21 22 am" src="https://user-images.githubusercontent.com/30979369/42183431-70177bc6-7e5f-11e8-8dce-8c02950b661f.png">

Reply:
And, it is genuine as well. dimen resource is used to keep dimensions consistent throughout different screen sizes. And, 0dp remains same for all screen sizes.

Reply:
0dp is fine